File: src/test/java/com/natu/ftax/ledger/domain/LedgerBookTest.java
package com.natu.ftax.ledger.domain;

import org.assertj.core.api.Assertions;
import org.junit.jupiter.api.Test;

class LedgerBookTest {
    @Test
    void testGenerateSimpleEntry() {
        LedgerBook ledgerBook = LedgerBook.create("1");
        LedgerEntry firstLedgerEntry = LedgerEntry.first("first");

        ledgerBook.getLedgerEntries().add(firstLedgerEntry);

        Assertions.assertThat(ledgerBook.getId()).isEqualTo("1");
        Assertions.assertThat(ledgerBook.getLedgerEntries().size()).isEqualTo(1);
    }
}

File: src/test/kotlin/com/natu/ftax/ledger/domain/LedgerBookTest.kt
package com.natu.ftax.ledger.domain


import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class LedgerBookTest {
    @Test
    fun testGenerateSimpleEntry() {
        val ledgerBook = LedgerBook.create("1")
        val firstLedgerEntry = LedgerEntry.first("first")

        ledgerBook.ledgerEntries.add(firstLedgerEntry)

        assertThat(ledgerBook.id).isEqualTo("1")
        assertThat(ledgerBook.ledgerEntries.size).isEqualTo(1)

    }


}
